src/Semaphore/MutexInterface.php

```diff
@@ -9,9 +9,20 @@ public function setIsBlocking(bool $isBlocking): MutexInterface;
 
     public function testAndSetLock(): bool;
 
+    /**
+     * Soft check for whether a mutex is available
+     *
+     * DOES NOT ACQUIRE THE MUTEX
+     *
+     * DON'T USE THIS UNLESS YOU KNOW WHAT YOU'RE DOING
```

Comment:
I think the name of the `testLock` method is clear enough about its behavior that this block comment is not needed. `test`, `set` and `testAndSet` are common actions when talking about locks.
```suggestion
```